app/forms/claimants_detail.rb

```diff
@@ -24,4 +26,19 @@ def to_h # rubocop:disable Metrics/MethodLength
       disagree_claimants_job_or_title: disagree_claimants_job_or_title
     }
   end
+
+  validates :claimants_name,
+    persons_name: true,
+    allow_blank: true
+  validates :agree_with_employment_dates, inclusion: { in: [true, false] }
+  validates :employment_start, :employment_end, :disagree_employment,
+    presence: true,
+    if: :disagree_with_employment_dates?
```

Comment:
You could use unless: agree_with_employment_dates if it reads better for you ?  You can then remove the disagree_with_employment_dates potentially

Reply:
Just gave it a go but I need that method anyway for the gov_uk_date_field validation, which only supports `validate_if` (ln 3).